DQMServices/Core/interface/ROOTFilePB.h

```diff
@@ -0,0 +1 @@
+#include "DQMServices/Core/src/ROOTFilePB.pb.h"
```

Comment:
@smuzaffar is this "trick" according to the rules? Isn't it the same as including from DQMServices/Core/src?
Why not to move directly DQMServices/Core/src/ROOTFilePB.pb.h into `\interface`?

Reply:
yes trick is according to the rules i.e. within a package one can use private headers :-) This allows unit tests of a package to use private headers and check the functionality.

I have no objections on moving `ROOTFilePB.pb.h` to `interface` too. We just need to make sure that next time when we update protobuf and re-generate thiese `ROOTFilePB.pb.*` files then the `ROOTFilePB.pb.h` should go in to `interface`.

Any objections @cms-sw/dqm-l2  ?

Reply:
No objections as far as I can tell

Reply:
> yes trick is according to the rules i.e. within a package one can use private headers :-) This allows unit tests of a package to use private headers and check the functionality.

Yes, but when included from DQMServices/FileIO/plugins/DQMFileSaverPB.cc or DQMServices/StreamerIO/plugins/DQMProtobufReader.cc it is not "within a package" any more...

Reply:
thats true @perrotta but build rules only look for direct include of a private header of other package. As `DQMServices/FileIO/plugins/DQMFileSaverPB.cc` is not directly including `src/ROOTFilePB.pb.h` so it will not fail. Anyway, I will update the PR to move ROOTFilePB.pb.h in to `interface`

Reply:
done, `src/ROOTFilePB.pb.h` moved to `interface/ROOTFilePB.pb.h`